app/Wiki.php

```diff
@@ -138,6 +138,8 @@ public function publicSettings()
             'wwWikibaseStringLengthMultilang',
             'wikibaseFedPropsEnable',
             'wikibaseManifestEquivEntities',
+            'wwUseQuestyCaptcha',
+            'wwCaptchaQuestions',
```

Comment:
I wonder if we should make this a blocklist instead? Else we'll be bitten by this again I surmise.

Reply:
Yeah, I guess things like wgSecret and the db credentials are things we would want to not expose to the manager but this list is fairly short.

Reply:
Today I learned there are DB credentials in `wikiSettings`. Then maybe let's keep it like this.